code/modules/client/preferences.dm

```diff
@@ -337,7 +337,7 @@ datum/preferences
 					else
 						dat += "[flavor_text]"
 				else
-					dat += "[copytext(flavor_text, 1, 37)]...<br>"
+					dat += "[TextPreview(flavor_text, 1, 37)]...<br>"

```

Comment:
Uh, this won't work. TextPreview only has two arguments, and this is setting len to 1.